LayoutTests/media/ios/media-sources-selection.html

```diff
@@ -0,0 +1,35 @@
+<html>
+<meta name="fuzzy" content="maxDifference=0-103; totalPixels=0-329652" /><head>
```

Comment:
Can we run this test for macOS as well?
Can we add a similar test for macOS + CRABS?

Reply:
The change only applies to macOS

Reply:
I added test for non-iOS ; to show that the first image is selected